lib/storage/providers/IDBKeyVal.js

```diff
@@ -102,7 +102,10 @@ const provider = {
      * @param {String} key
      * @return {Promise<*>}
      */
-    getItem: key => get(key, getCustomStore()),
+    getItem: key => get(key, getCustomStore())
+
+        // idb-keyval returns undefined for missing items, but this needs to return null.
```

Comment:
```suggestion
        // idb-keyval returns undefined for missing items, but this needs to return null so that idb-keyval does the same thing as SQLiteStorage.
```